doc/javadoc.hpp

```diff
@@ -10,42 +10,63 @@
 /** {brief}
 
     {description}
+    <!-- Assume the user knows C++ -->
+    <!-- Assume the user read the whole exposition -->
```

Comment:
read or "will read"

Reply:
I meant "read" in the past tense. Or a user who "has read" the whole exposition. 

I inferred this rule in the last PR, as you told me to assume in the reference API, the user has read the whole exposition.

Reply:
well that's why I say "has read" _or_ "will read"

Reply:
Exactly. I've changed it to "has read". The present tense would be "reads" here, so it still wouldn't be ambiguous, but "has read" is better.